homework8_max_3.py

```diff
@@ -0,0 +1,35 @@
+"""
+Homework8.
+
+Description:
+    Given the list of integers ( positive , negative, zeros)
+    Find max multiplication of three values in list
+    l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+    Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+    Output: Max value = "360". Nums are: (12, 5, 6)
+
+    Input: [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+    Output: Max value = "2016". Nums are: (-7, 12, -24)
+
+"""
+
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+# create a list where each element is squared
+converted = [num**2 for num in l1]
+
+# sort list
+sorted_list = sorted(converted, reverse=True)
```

Comment:
Please check your solution vs 
l1 = [-7, 0, 1, 16, 24]
Max value_l1 = 384. Nums are: (24,16,1)

In you case 
Max value = "-2688". Nums are: [24, 16, -7]
384 > -2688   Maximum should be 384

Reply:
reworked in other way